package.json

```diff
@@ -50,12 +50,16 @@
     "ipfs": "^0.33.1",
     "ipfs-http-client": "^28.0.0",
     "node-localstorage": "^1.3.1",
+    "uid": "0.0.2",
     "varuint-bitcoin": "^1.1.0",
     "wif": "^2.0.6"
   },
   "files": [
     "lib"
   ],
+  "publishConfig": {
+    "registry": "https://mediciventures.jfrog.io/mediciventures/api/npm/mlg-npm/"
```

Comment:
Don't commit change to private MLG npm registry ;)

Similarly for the changed name/version at top of file